Portal/src/Datahub.Portal/Layout/Error.razor

```diff
@@ -74,10 +73,23 @@
 
     private bool _emailSent;
 
+    private string _supportLink;
+
+
     private MarkupString GetGeneralErrorText()
     {
         var template = Localizer["An application error has occurred on the DataHub server. The error has been recorded with Correlation ID {0}. To avoid network issues, if you are currently on the VPN, please try to disconnect the VPN and retry your actions."].ToString();
```

Comment:
This text is missing for i18n, appearing in English on the French page

Reply:
Thanks! cant believe i missed that one.